.ci/check-python-dists.sh

```diff
@@ -16,11 +16,11 @@ pip install \
     twine || exit 1
 
 echo "twine check..."
-twine check --strict ${DIST_DIR}/* || exit 1
+twine check --strict "${DIST_DIR}"/* || exit 1
 
 if { test "${TASK}" = "bdist" || test "${METHOD}" = "wheel"; }; then
     echo "check-wheel-contents..."
-    check-wheel-contents ${DIST_DIR}/*.whl || exit 1
+    check-wheel-contents "${DIST_DIR}"/*.whl || exit 1
```

Comment:
```suggestion
    check-wheel-contents "$(echo "${DIST_DIR}/"*.whl)" || exit 1
```

Reply:
In the future, please click "add suggestion to batch" and then eventually "commit suggestions" on these inline suggestion comments. That way, reviewers won't have to manually verify that you've applied the suggested changes, and GitHub will automatically mark the conversation "resolved" and collapse the threads.